src/errors.js

```diff
@@ -1,28 +1,51 @@
 'use strict'
 
 const errcode = require('err-code')
-
-module.exports.dbOpenFailedError = (err) => {
+/**
+ *
+ * @param {Error} [err]
```

Comment:
Why aren't we documenting return types?

Reply:
no specific reason, in these situations i just let TS infer from the calls. When the code is more complex and i want to make sure the return type is what i want i explicitly write the return type.